Tests/Virtual Time/VirtualTimers.swift

```diff
@@ -12,46 +12,19 @@ struct VirtualTimeTimer: StreamChatClient.Timer {
     
     static var time: VirtualTime!
     
-    static func schedule(timeInterval: DispatchTimeInterval, queue: DispatchQueue, onFire: @escaping () -> Void) {
-        _ = Self.time.scheduleTimer(interval: TimeInterval(dispatchTimeInterval: timeInterval),
+    static func schedule(timeInterval: TimeInterval, queue: DispatchQueue, onFire: @escaping () -> Void) {
+        _ = Self.time.scheduleTimer(interval: timeInterval,
                                     repeating: false,
                                     callback: { _ in onFire() })
     }
     
-    static func scheduleRepeating(timeInterval: DispatchTimeInterval,
+    static func scheduleRepeating(timeInterval: TimeInterval,
                                   queue: DispatchQueue,
                                   onFire: @escaping () -> Void) -> TimerControl {
-        Self.time.scheduleTimer(interval: TimeInterval(dispatchTimeInterval: timeInterval),
+        Self.time.scheduleTimer(interval: timeInterval,
                                 repeating: true,
                                 callback: { _ in onFire() })
     }
 }
 
-extension VirtualTime.Timer: TimerControl {
-    func resume() {
-        isActive = true
-    }
-    
-    func suspend() {
-        isActive = false
-    }
-}
-
-extension TimeInterval {
-    init(dispatchTimeInterval: DispatchTimeInterval) {
-        switch dispatchTimeInterval {
-        case .seconds(let value):
-            self = Double(value)
-        case .milliseconds(let value):
-            self = Double(value) / 1_000
-        case .microseconds(let value):
-            self = Double(value) / 1_000_000
-        case .nanoseconds(let value):
-            self = Double(value) / 1_000_000_000
-        case .never:
-            self = .greatestFiniteMagnitude
-        @unknown default:
-            fatalError()
-        }
-    }
-}
+extension VirtualTime.TimerControl: TimerControl { }
```

Comment:
Why don't we put it in `VirtualTime.swift` file, or directly to implementation?

Reply:
`VirtualTime.swift` is a file independent to `StreamChatClient`. You can take it and include to any target you want. `VirtualTimers.swift` is where `VirtualTime` is used to implement internal protocols `StreamChatClient .Timer` and `StreamChatClient.TimerControl`. It's good to have them separate.